entries/c/cloudamqp.com.json

```diff
@@ -0,0 +1,17 @@
+{
+  "CloudAMQP": {
+    "domain": "cloudamqp.com",
+    "img": "cloudamqp-logo.png",
+    "tfa": [
+      "totp",
+      "sms"
+      
+    ],
+    "documentation": "https://www.cloudamqp.com/blog/two-factor-authentication.html",
+    "notes": "Sms is required as a backup method",
+    "keywords": [
+      "cloud",
```

Comment:
From what I understand RabbitMQ is a message-broker and not cloud computing so the cloud category should be removed.
```suggestion
```